README.md

```diff
@@ -0,0 +1,72 @@
+# bunyan-env-config
+
+We want good logs.  The debug module is great, but it's a little limited for
+the size of our current set of taskcluster services.  We've decided to move our
+components to use something a little more powerful.  Our choice is to use the
+bunyan module with a small wrapper.  The wrapper takes care of importing bunyan
+and creating the root logger.  The logger returned by the setup method is a
+standard bunyan root logger with the exception of a debugCompat property added.
```

Comment:
So it's not really different to raw bunyan.  This library is just a wrapper of bunyan to simplify configuration and to allow a debug compatibility module to be created.  I can change the text to reflect that.